go.mod

```diff
@@ -6,7 +6,7 @@ require (
 	github.com/Microsoft/go-winio v0.4.14
 	github.com/cenkalti/hub v1.0.1-0.20140529221144-7be60e186e66 // indirect
 	github.com/cenkalti/rpc2 v0.0.0-20140912135055-44d0d95e4f52 // indirect
-	github.com/contiv/libOpenflow v0.0.0-20200728044739-7c6534390721
+	github.com/contiv/libOpenflow v0.0.0-20201014051314-c1702744526c
```

Comment:
Is this upgrade by intention? what change it brings?

Reply:
There are breaking changes libOpenflow which will make the compile failed if we don't update here.